src/rdkafka.h

```diff
@@ -1018,6 +1018,26 @@ void rd_kafka_conf_set_events(rd_kafka_conf_t *conf, int events);
 
 
 /**
+ * @brief Creates/configures the SSL context using the given callback
```

Comment:
"Creates": the context has already been created and setup by librdkafka, so this callback can only provide additional configuration (or perhaps replace previously set config?).

I'm also wondering if maybe this should be an interceptor instead, allowing multiple sources (plugins, app) to modify the CTX.